css/index.css

```diff
@@ -143,4 +236,107 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/*My stuff*/
+
+header {
+    display: flex;
+}
+
+header img {
+    margin-right: 50px;
+}
+
+header nav {
+    display: flex;
+    flex: 1;
+    justify-content: space-between;
+    align-items: flex-end;
+    padding: 0 20px;
+}
+
+header nav a {
+    color: black;
+    text-decoration: none;
+}
+
+.jumbo {
+    padding: 40px 0;
+}
+
```

Comment:
Good job using the parent styles to style the boxes and only overriding the color for each box.